contracts/RLPReader.sol

```diff
@@ -10,13 +10,39 @@ library RLPReader {
     uint8 constant LIST_SHORT_START   = 0xc0;
     uint8 constant LIST_LONG_START    = 0xf8;
 
+    uint constant STRING_LONG_OFFSET = 0xb7;
```

Comment:
these do not seem to be used. Can you remove thsee

Reply:
Sure.